src/components/Reorder/Group.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from "react"
+import { FunctionComponent, ReactHTML, useEffect, useRef } from "react"
+import { ReorderContext } from "../../context/ReorderContext"
+import { motion } from "../../render/dom/motion"
+import { HTMLMotionProps } from "../../render/html/types"
+import { useConstant } from "../../utils/use-constant"
+import { ItemData, ReorderContextProps } from "./types"
+import { checkReorder } from "./utils/check-reorder"
+
+interface Props<V> {
```

Comment:
Why don't we use the `RenderGroupProps` type directly here?

Reply:
I should have removed `RenderGroupProps`